src/utils/timestamp-utils.ts

```diff
@@ -0,0 +1,44 @@
+import { Timestamp } from "@hashgraph/sdk";
+import moment from 'moment';
+
+export class TimestampUtils {
+    public static ISO = "YYYY-MM-DDTHH:mm:ss.SSS[Z]";
+    public static ISO8601 = "YYYY-MM-DDTHH:mm:ss[Z]";
+
+    public static toJSON(item: Timestamp, format: string = this.ISO): string {
```

Comment:
toJSON and fromJson. What capitalization strategy should we use?

Reply:
'toJSON' is javascript standart method redefinition. 'fromJson' and others come from did-sdk-java for docs compatability